RecoEgamma/EgammaTools/interface/AnyMVAEstimatorRun2Base.h

```diff
@@ -12,19 +12,15 @@ class AnyMVAEstimatorRun2Base {
 
  public:
   // Constructor, destructor
- AnyMVAEstimatorRun2Base(const edm::ParameterSet& conf) : _conf(conf) {}
+ AnyMVAEstimatorRun2Base(const edm::ParameterSet& conf) : conf(conf) {}
```

Comment:
Out of curiosity: is there any possible ambiguity from this line?
@Dr15Jones ?

Reply:
If it compilers, there is no confusion :). However the member data name `conf` does violate CMS naming policy.

Reply:
I thought only private members are named with a training underscore. Ok, next time also the public ones. But `conf` it should actually be private, I just didn't want to change too much.